src/main.cpp

```diff
@@ -203,9 +204,23 @@ std::map<uint256, NodeId> mapBlockSource;
  *
  * Memory used: 1.7MB
  */
-boost::scoped_ptr<CRollingBloomFilter> recentRejects;
+boost::scoped_ptr<CRollingBloomFilter> recentRejects GUARDED_BY(cs_recentRejects);
```

Comment:
why not using `std::unique_ptr` like we did below for `txn_recently_in_block`?

Reply:
true, i'll change that.